### PR TITLE
updated gomplate version and added ppc64le support

### PR DESCRIPTION
**Pull request by mayurwaghmode**
Signed-off-by: mayurwaghmode <waghmodemayur17@gmail.com>

#### This PR has updated the gomplate version to the latest release(v3.11.2) and added ppc64le support to the dex binary and image. Gomplate's latest release supports ppc64le architecture as well- 
 https://github.com/hairyhenderson/gomplate/releases/tag/v3.11.2

### CODE REVIEW

**Comment by mayurwaghmode** [comment]
@nabokihms @sagikazarmark can u please review this PR?

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le
```
Comment (nabokihms) [inline .github/workflows/docker.yaml:78]: https://github.com/dexidp/dex/blob/e74acdff6c68c00029360709c553d301a7850b5f/.github/workflows/artifacts.yaml#L73

The new architecture should also be added to the artifacts build job.

**Review by nabokihms — CHANGES_REQUESTED** [review]
@mayurwaghmode Looks good to me besides the one small thing. 

A question should also be asked: have you tested the image for ppc64le somewhere?

File: .github/workflows/docker.yaml (hunk 1 above)
Comment (mayurwaghmode) [reply .github/workflows/docker.yaml:78]: updated

**Comment by mayurwaghmode** [comment]
> @mayurwaghmode Looks good to me besides the one small thing.
> 
> A question should also be asked: have you tested the image for ppc64le somewhere?

Yes I have tested it on ppc64le architecture
```
[root@unison1 ~]# arch
ppc64le
[root@unison1 ~]# podman run ghcr.io/mayurwaghmode/dex:latest
Trying to pull ghcr.io/mayurwaghmode/dex:latest...
Getting image source signatures
Copying blob 98ac821467a7 done
Copying blob 5f9f35a7afcc done
Copying blob 43b55e722196 done
Copying blob 875d3eebdd28 done
Copying blob 57def83ab84a done
Copying blob d3deabf2a506 done
Copying blob 7e222264ff50 done
Copying blob 20519a839c09 done
Copying blob 61210fa3cc0c done
Copying blob 7d5206fadad4 done
Copying config 771e854100 done
Writing manifest to image destination
Storing signatures
time="2022-08-11T08:57:03Z" level=info msg="Dex Version: 4ebdaab1427d5ba664ba6f0244dd6c512a4f9b57-dirty, Go Version: go1.18.3, Go OS/ARCH: linux ppc64le"
```

**Review by nabokihms — APPROVED** [review]
@mayurwaghmode looks good to me, thank you for contributing!

**Pull request merged**